modules/custom/az_publication/az_publication.links.task.yml

```
@@ -33,17 +33,39 @@ entity.az_author.collection:
 
 az_publication.az_publication_settings_tab:
   route_name: az_publication.settings
-  title: 'AZ Publication'
+  title: 'Publication'
```

I think it would be more consistent to keep the `AZ` convention here to match the other custom module setting tabs.
```suggestion
  title: 'AZ Publication'
```